modules/__tests__/LocationUtils-test.js

```diff
@@ -111,4 +111,123 @@ describe('createLocation', () => {
       })
     })
   })
+
+  describe('with a current location', () => {
+    const currentLocation = {
+      pathname: '/The/Old/Path',
+      search: '?theQuery',
+      hash: '#fragment'
+    };
+
+    const createRelative = url => createLocation(url, undefined, undefined, currentLocation)
```

Comment:
It's ok to leave `currentLocation` up top. I just like to have some consistency between different tests.